core/src/main/java/bisq/core/setup/CoreNetworkCapabilities.java

```diff
@@ -29,7 +29,7 @@
 public class CoreNetworkCapabilities {
     public static void setSupportedCapabilities(BisqEnvironment bisqEnvironment) {
         Capabilities.app.addAll(Capability.TRADE_STATISTICS, Capability.TRADE_STATISTICS_2, Capability.ACCOUNT_AGE_WITNESS, Capability.ACK_MSG);
-        Capabilities.app.addAll(Capability.BUNDLE_OF_ENVELOPES);
+        Capabilities.app.addAll(Capability.BUNDLE_OF_ENVELOPES,Capability.SIGNED_ACCOUNT_AGE_WITNESS);
```

Comment:
not sure if that line is safe. maybe not add the capability for now?

Reply:
No issues, this is the way to extend the protocol. We're saying we handle `SIGNED_ACCOUNT_AGE_WITNESS` but no one is going to ask for it until we add the code actually using it.